draft-ietf-jsonpath-base.md

```diff
@@ -682,6 +682,19 @@ and produces a nodelist consisting of that root node.
 
 ### Examples
 
+<aside markdown="1">
+In this and the following examples in Sections {{<root-identifier}} and
+{{<selector-details}} except for {{tbl-comparison}}, we will present a
+JSON text to show the JSON value used as the query argument to the
+queries in the examples, and then a table with the columns:
+
+* Query: an example query to be applied to the query argument
+* Result: the query result as a list of JSON values that were located in the query argument
+* Result Path: the query result as a list of (normalized) paths into
+  the query argument, giving locations in parallel to the previous column
```

Comment:
```suggestion
  the query argument, giving locations of the JSON values in the previous column
```

Reply:
Do we want a full stop at the end of the last bullet?

Reply:
> Do we want a full stop at the end of the last bullet?

Not normally.  We either use punctuation throughout the list or leave it off entirely.
There are no sentences here, so the latter is the usual choice.